### PR TITLE
Simplify/correct a few examples

### PR DESCRIPTION
**Pull request by isiahmeadows**
- `Array.prototype.slice.call(arrayLike).forEach(func)` -> `Array.prototype.forEach.call(arrayLike, func)`
- `Array.prototype.concat.apply([], ...args)` -> `Array.prototype.concat.apply([], arguments)`
- etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -966,8 +966,8 @@ Most of utilities are found by native API. Others advanced functions could be ch
 
   // Native
   // But concat function doesn't remove duplicate items.
-  function merge(...args) {
```
Comment (softwarespot) [inline README.md]: I would still like to retain ES2015 rest params over arguments, since arrow functions etc... are used throughout.

**Comment by tracker1** [comment]
I wouldn't mind seeing a move towards using `Array.from` (with a note to polyfill) over some of the various call/apply statements.

File: README.md (hunk 1 above)
Comment (isiahmeadows) [inline README.md]: Done.

**Comment by isiahmeadows** [comment]
@tracker1 I wouldn't mind, either, but this declares its baseline on IE10+ (which has practically zero ES6 support IIRC), so that's why I didn't. If we dealt with modern APIs, this list would be very different.

File: README.md (hunk 1 above)
Comment (softwarespot) [inline README.md]: Thanks

**Pull request merged**